messenger/http_transporter.go

```diff
@@ -181,6 +181,14 @@ func (t *HTTPTransporter) Install(msgName string) {
 func (t *HTTPTransporter) Listen() error {
 	host := t.upid.Host
 	port := t.upid.Port
+
+	if host == "" {
+		host = "0.0.0.0"
+	}
+
+	if port == "" {
+		port = "0"
```

Comment:
is port 0 really valid? shouldn't the default be 5051 for slaves and 5050 for master nodes?